app/router.js

```diff
@@ -64,4 +64,5 @@ Router.map(function () {
 
   this.route('team-redirect', { path: 'team' });
   this.route('teams');
+  this.route('ember-community-survey-2021');
```

Comment:
Since we're adding a route, I need to ask for a few things:

1. I want to keep the list of routes in alphabetical order to help with maintenance. Can you move this line to after line 34 (`ember-community-survey-2020`)?

2. Will you also add `ember-community-survey-2021` to `ember-cli-build.js` file, under `prember`? The list is also in alphabetical order. (https://github.com/ember-learn/ember-website/blob/master/ember-cli-build.js#L58-L59)

3. Will you add an application test file named `ember-community-survey-2021-test.js`? You can copy-paste the 2020's test file to create a Percy snapshot and run accessibility checks for 2021 page.

Reply:
Will do!